.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,128 @@
+name: Publish Docker Image
+
+on:
+  push:
+    branches:
+      - main
+      - v1.16
+    tags:
+      - v1.*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      REPOSITORY: ${{ secrets.DOCKER_HUB_ORGS }}/fluentd-kubernetes-daemonset
+    strategy:
+      fail-fast: false
+      matrix:
+        context:
+          - docker-image/v1.16/arm64/debian-azureblob
+          - docker-image/v1.16/arm64/debian-cloudwatch
+          - docker-image/v1.16/arm64/debian-elasticsearch7
+          - docker-image/v1.16/arm64/debian-elasticsearch8
+          - docker-image/v1.16/arm64/debian-forward
+          - docker-image/v1.16/arm64/debian-gcs
+          - docker-image/v1.16/arm64/debian-graylog
+          - docker-image/v1.16/arm64/debian-kafka
+          - docker-image/v1.16/arm64/debian-kafka2
+          - docker-image/v1.16/arm64/debian-kinesis
+          - docker-image/v1.16/arm64/debian-logentries
+          - docker-image/v1.16/arm64/debian-loggly
+          - docker-image/v1.16/arm64/debian-logzio
+          - docker-image/v1.16/arm64/debian-opensearch
+          - docker-image/v1.16/arm64/debian-papertrail
+          - docker-image/v1.16/arm64/debian-s3
+          - docker-image/v1.16/arm64/debian-syslog
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Setup tags
+        run: |
+          set -x
+          context=${{ matrix.context }}
+          component=${context##*/}
+          for target in $(make echo-all-images); do \
+            case $target in \
+              *$component-amd64*) \
+                tags=$(echo $target | cut -d':' -f2-); \
+                tag1=$(echo $tags | cut -d',' -f1); \
+                tag2=$(echo $tags | cut -d',' -f2); \
+                echo "AMD64TAGS=${{ env.REPOSITORY }}:${tag1},${{ env.REPOSITORY }}:${tag2}" >> ${GITHUB_ENV}; \
+                echo "MULTIARCH_TAG=${tag1/amd64-/}" >> ${GITHUB_ENV}; \
+                echo "MULTIARCH_SHORT_TAG=${tag2/amd64-/}" >> ${GITHUB_ENV}; \
+                echo "AMD64TAG=${tag1}" >> ${GITHUB_ENV}; \
+                echo "SHORT_AMD64TAG=${tag2}" >> ${GITHUB_ENV}; \
+                ;; \
+              *$component-arm64*) \
+                tags=$(echo $target | cut -d':' -f2-); \
+                tag1=$(echo $tags | cut -d',' -f1); \
+                tag2=$(echo $tags | cut -d',' -f2); \
+                echo "ARM64TAGS=${{ env.REPOSITORY }}:${tag1},${{ env.REPOSITORY }}:${tag2}" >> ${GITHUB_ENV}; \
+                echo "ARM64TAG=${tag1}" >> ${GITHUB_ENV}; \
+                echo "SHORT_ARM64TAG=${tag2}" >> ${GITHUB_ENV}; \
+                ;; \
+            esac \
```

Comment:
The `\` or `; \` at the end of these lines would not be necessary.
No line continues to the next line or combines multiple lines.
I have tested the following sample code:

* Code: https://github.com/daipom/fluentd-kubernetes-daemonset/blob/test/.github/workflows/docker-build.yml
* Result: https://github.com/daipom/fluentd-kubernetes-daemonset/actions/runs/9638976525/job/26580483264

Reply:
Thanks, fixed it.